robot-core/src/test/java/org/obolibrary/robot/ReasonerHelperTest.java

```diff
@@ -49,7 +49,9 @@ public void testIncoherentRBox()
       } catch (IncoherentRBoxException e) {
         isCaughtException = true;
       }
-      assertTrue(isCaughtException);
+      // To our great disappointment, ELK does not recognise this incoherent R Box
+      // https://github.com/ontodev/robot/pull/999#issuecomment-1666567017
+      // assertTrue(isCaughtException);
```

Comment:
@jclerman @balhoff to my great suprise, ELK 0.5 does not work with this recently introduced probing trick that @jclerman added. I checked it in Protege. It truly does not recognise it. Waiting for an Elk 0.5 fix is IMO out of the question: https://github.com/liveontologies/elk-reasoner/issues/66

Reply:
Solved, see below.